components/locale_canonicalizer/src/locale_canonicalizer.rs

```diff
@@ -579,16 +579,19 @@ impl<'data> LocaleCanonicalizer<'data> {
 
         let mut max = langid.clone();
         self.maximize(&mut max);
+        let variants = max.variants.clone();
         max.variants.clear();
```

Comment:
`nit`: Please, use `let variants = max.variants.drain(..).collect()` to avoid copying.

Reply:
sg!

Reply:
@zbraniecki I am not able to use `let variants = max.variants.drain(..).collect()`. The error is 
```
let variants = max.variants.drain(..).collect();
    |                                     ^^^^^ method not found in `Variants`
```

I am new to Rust, not sure how to fix it. But it appears that `max.variants` is not a vec, it is a Box pointer.